core/Decode.cpp

```diff
@@ -76,24 +130,66 @@ namespace olympia
         uint32_t num_decode = std::min(uop_queue_credits_, fetch_queue_.size());
         num_decode = std::min(num_decode, num_to_decode_);
 
+        //buffer to maximize the chances of a group match limited
+        //by max allowed latency, bounded by max group size
+        if(fusion_enable_) {
+          if(num_decode < fusion_max_group_size_
+             && latency_count < fusion_max_latency_)
+          {
+            ++latency_count;
+            return;
+          }
+        }
```

Comment:
nit: tabbing is off

Reply:
I will run clang-format on Decode.cpp and .hpp. That will fix this. 

There will likely be unrelated format changes.